tests/models/falcon/test_falcon.py

```diff
@@ -1,25 +1,37 @@
 import torch
-import pytest
 
 # Load model directly
-from transformers import AutoTokenizer, AutoModelForCausalLM
+from transformers import AutoTokenizer, AutoModelForCausalLM, AutoConfig
 
 
-@pytest.mark.xfail
 def test_falcon(record_property):
     record_property("model_name", "Falcon")
 
     # Download model from cloud
     model_name = "tiiuae/falcon-7b-instruct"
+
+    # Falcon hidden_size is 4544, which is too big to fit into layer_norm
+    config = AutoConfig.from_pretrained(model_name)
+    config.hidden_size = config.hidden_size // 2
```

Comment:
I think we should avoid such changes to models.
We should target to intake a model with 0 modifications. 

Think that any modification is a "fine".

If model does not fit - ideally compiler figures that input should be split.

@boris-drazic @mcw-zwakeelTT here is an example of input and op which require splitting.

Reply:
If we are to modify a model like that, lets at least always keep the original model with 0 modifications. 
Our target workflow - user takes a model, adds compile and it just runs

Reply:
Since is the default value for this pretrained model, I'll revert this and mark it as xfail again. I have an issue https://github.com/tenstorrent/pytorch2.0_ttnn/issues/63 to track and make it easier to test this scenario.